extension/src/pipeline/quickPick.test.ts

```diff
@@ -47,39 +49,51 @@ const mockValidData = [
 
 describe('pickPlotConfiguration', () => {
   it('should let the user pick from files with accepted data types', async () => {
-    mockedPickFile.mockResolvedValueOnce('file.json')
-    mockedLoadDataFile.mockReturnValueOnce(mockValidData)
+    mockedPickFiles.mockResolvedValueOnce(['file.json'])
+    mockedLoadDataFiles.mockResolvedValueOnce([
+      { data: mockValidData, file: 'file.json' }
+    ])
 
-    await pickPlotConfiguration()
+    await pickPlotConfiguration('/')
 
-    expect(mockedPickFile).toHaveBeenCalledWith(Title.SELECT_PLOT_DATA, {
+    expect(mockedPickFiles).toHaveBeenCalledWith(Title.SELECT_PLOT_DATA, {
       'Data Formats': ['json', 'csv', 'tsv', 'yaml']
     })
   })
 
   it('should return early if user does not select a file', async () => {
-    mockedPickFile.mockResolvedValueOnce(undefined)
+    mockedPickFiles.mockResolvedValueOnce(undefined)
 
-    const result = await pickPlotConfiguration()
+    const result = await pickPlotConfiguration('/')
 
     expect(result).toStrictEqual(undefined)
   })
 
-  it('should show a toast message if the file fails to parse', async () => {
-    mockedPickFile.mockResolvedValueOnce('file.csv')
-    mockedLoadDataFile.mockReturnValueOnce(undefined)
+  it('should show a toast message if the files are not the same data type', async () => {
+    mockedPickFiles.mockResolvedValueOnce(['file.json', 'file.csv'])
```

Comment:
[Q] Is this a dvc constraint?

Reply:
Yes, DVC needs the files to be the same type.